liblwgeom/lwprint.c

```diff
@@ -378,6 +378,8 @@ static char * lwdouble_to_dms(double val, const char *pos_dir_symbol, const char
 
 	/* Allocate space for the result.  Leave plenty of room for excess digits, negative sign, etc.*/
 	result = (char*)lwalloc(format_length + WORK_SIZE);
+	memset(result, 0, format_length + WORK_SIZE);
```

Comment:
Do you think we should add a `lwcalloc` to `lwgeom` ?

Reply:
I've thought about that and it might help, but at the end of of the day if you don't use `memset` you probably won't use `lwcalloc` either.

Also, this case looks like a false positive as I don't see any way for the memory outside of the set string to be read, but since it's an output function and memset shouldn't any noticeable impact in performance I've decided to set it and make the sanitizer output clearer.

Reply:
` grep -R -B1 "memset(" --include "*.c" | grep alloc | wc` shows 20 or so usages. Seems worth it to me.

Reply:
Most of those seem uses to set simple structs, but I wouldn't oppose for it to be added in a followup issue.